release-notes/General/General_Feature_Release_10.3/General_Feature_Release_10.3.6.md

```diff
@@ -15,7 +15,103 @@ uid: General_Feature_Release_10.3.6
+
+##### Configuring the UserDefinableApiEndpoint extension module
+
+The first time you use DataMiner User-Defined APIs, make sure you adjust the UserDefinableApiEndpoint DxM settings to match your API needs.
+
+To do so, go to the folder `%programfiles%\Skyline Communications\DataMiner UserDefinableApiEndpoint\`, and create a copy of the file *appsettings.json* with the name *appsettings.custom.json*. You can then create your custom settings in that new file.
+
+These are the main settings you can adjust:
+
+- *Kestrel* > *Endpoints* > *Http* > *Url*: This setting contains the endpoint where the UserDefinableApiEndpoint DxM will be run. This is where you can customize which port is used. Note that IIS also has a rewrite rule (Reroute User Definable APIs) that forwards API requests to the port used by UserDefinableApiEndpoint (5002). When you specify a custom port, you will also have to update that rule.
+- *Kestrel* > *Limits* > *MaxConcurrentConnections*: This setting defines the maximum number of open connections. By default, the DxM will allow 100 concurrent connections, but you can customize this number here.
+- *Serilog*: Serilog is the logging service used for UserDefinableApiEndpoint. Here you can change where the log files should be located, how big they can get, and how many files should be kept. You can also change the default log levels.
+- *UserDefinableAPIs* > *MessageBrokerTimeOutSeconds*: This setting indicates how long the message broker (sending the NATS trigger to SLNet) will wait for a response before it times out. By default, this is set to 90 seconds (i.e. 1.5 minutes). If you increase the timeout value, you will also need to increase the timeout in IIS.
+
+> [!TIP]
+> For detailed information and examples, see [Configuring the DxM](xref:UD_APIs_UserDefinableApiEndpoint#configuring-the-dxm).
+
+##### Defining an API
+
+To define an API, you will first need to **create an Automation script** containing the logic of the API. This Automation script should contain the OnApiTrigger entry point method, which will be executed when the API is triggered. Alternatively, you can also **use an existing script** and trigger the API through the *Run* method. However, the latter approach has some disadvantages: you will not have access to the *ApiTriggerInput* object and *ApiTriggerOutput* object in the script, and it will therefore not be possible to check the route, request the method of the API trigger, or output anything back to the API caller.
+
+When the script is ready, you will then need to **create an API and tokens**. This can be done in the Automation module, or directly in code. If you use DataMiner Automation, open the script and click *Configure API* at the bottom of the window. This will open a window where you can specify the following information:
```

Comment:
This can be done in the Automation module or in System Center. In the User-Defined APIs tab in System center,  you can create, edit and delete APIs and tokens. Maybe it's not needed here to mention that it can be done directly in code? (because users should mainly do it via the UI)